Resources/Prototypes/Nuclear14/Entities/Objects/Weapons/Guns/Rifles/rifles.yml

```diff
@@ -43,3 +43,141 @@
     steps: 1
     zeroVisible: true
   - type: Appearance
+
+- type: entity
+  name: Assault Carbine
+  parent: BaseWeaponRifle
+  id: WeaponCarbine
```

Comment:
New entities should have N14 at the start so they're easier to search and if possible narrow down the type of weapon.

e.g. Change this to `N14WeaponRifleCarbine`